airflow/www_rbac/views.py

```diff
@@ -147,6 +147,33 @@ def render(self, template, **context):
 
 
 class Airflow(AirflowBaseView):
+    @expose('/health')
+    @provide_session
+    def health(self, session=None):
+        from sqlalchemy import func
```

Comment:
why do local import instead of global import for the function? Do we have namespace conflict?